modules/alloy/src/main/ts/ephox/alloy/behaviour/positioning/ActivePosition.ts

```diff
@@ -5,7 +5,11 @@ import { PositioningConfig } from './PositioningTypes';
 const exhibit = (base: DomDefinitionDetail, posConfig: PositioningConfig): DomModification.DomModification =>
   DomModification.nu({
     classes: [ ],
-    styles: posConfig.useFixed() ? { } : { position: 'relative' }
+    styles: {
```

Comment:
This is pretty difficult to follow and a bit risky. I think now that we can't use a ternary because there's more than 2 things, because as a general rule we avoid nested ternaries, we should just use `if / else` etc. Here, if `usePositioningType` has an invalid value (which is definitely possible because boulder types gets coerced), then this would be undefined. So something like: 

```
const getStyles = (positioningType: 'fixed' | 'relative' | 'absolute'): Record<string, string> => {
  if (...) {
    return ...
  } else if (...) {
  
  } else {
    return { }
  }
};
```
might be better.

Reply:
Refactored [here](https://github.com/tinymce/tinymce/pull/8294/commits/a49621cb68f9160ca0a31e740078186ae89718a8).